packages/ui-rating/src/RatingIcon/__tests__/RatingIcon.test.tsx

```diff
@@ -26,9 +26,8 @@ import React from 'react'
 import { expect, mount, within } from '@instructure/ui-test-utils'
 
 import { InstUISettingsProvider } from '@instructure/emotion'
-// @ts-expect-error ts-migrate(7016) FIXME: Could not find a declaration file for module '@ins... Remove this comment to see the full error message
-// eslint-disable-next-line no-restricted-imports
-import { TransitionLocator } from '@instructure/ui-motion/es/Transition/TransitionLocator'
```

Comment:
Hm, I1m not sure about this, might be a reason why they use the built version here. @matyasf or @Brailor you guys have any idea?

Reply:
ping @matyasf és @Brailor :)

Reply:
hmm I don't think it has any special meaning, I think it will get this file from `/es/` by default. And if something would be wrong it would fail with some kind of "package not found" error.

Reply:
I would put it back as it was, this really seems like sth that was deliberately made this way. Later we can have a look at these and fix everywhere, but I wouldn't risk breaking sth as part of a TS typing task :)

Reply:
Please add this back as it was :)

Reply:
I think this is the only thing left from this PR